core/src/test/java/org/elasticsearch/index/query/RangeQueryBuilderTests.java

```diff
@@ -270,7 +330,7 @@ public void testDateRangeBoundaries() throws IOException {
                 "        }\n" +
                 "    }\n" +
                 "}\n";
-        Query parsedQuery = parseQuery(query).toQuery(createShardContext()).rewrite(null);
```

Comment:
for my education, why are the rewrite calls gone here? or maybe the right question is why we were calling rewrite before?